src/Form/Type/ImmutableTabsType.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+/*
+ *
+ * This file is part of the Sonata for Ekino project.
+ *
+ * (c) 2018 - Ekino
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ */
+
+namespace Sonata\SonataHelpersBundle\Form\Type;
```

Comment:
Is it the right namespace?

Reply:
the  one configured in composer.json
`    "psr-4": { "Sonata\\SonataHelpersBundle\\": "src/" }
`

Reply:
Ok, in that case I think we should differentiate bundle & component, to be easily compatible with symfony 4, what do you think @bmaziere ?

Reply:
i agree. we will separate bundle stuff from component stuff. but this one is a bundle stuff and there is no issue here to be compatible symfony4 isn't it?

Reply:
If I understand this correctly, there are no more bundles in SF4: https://medium.com/@fabpot/symfony-4-monolith-vs-micro-52dc6b98c0c5 (doesn't mean that it's not compatible, but at least not best practice)

Reply:
no more bundle in your main application but still third party bundle from what i understood no? i may be out of date ;)

Reply:
Yes Bundle are third party, only the main app is bundleless

Reply:
so will this be the bundle repo or the component repo ?